lms/static/js/inline_analytics.js

```diff
@@ -378,6 +371,23 @@ window.InlineAnalytics = (function() {
     }
 
 
+    function convertNameToArray(choiceNameString) {
+        // Convert the choice name string to a list
+        choiceNameString = choiceNameString.substring(1, choiceNameString.length -1).replace(/'/g, "");
```

Comment:
What does the value of `choiceNameString` look like?

Is it worth giving an example in a comment on this function?

```javascript
function convertNameToArray(choiceNameString) {
    // choiceNameString looks like `xstv's choice`
    // ...
}